GoogleDataTransport/GDTCCTLibrary/GDTCCTUploadOperation.m

```diff
@@ -692,11 +564,12 @@ - (void)URLSession:(NSURLSession *)session
     return;
   }
   if (response.statusCode == 302 || response.statusCode == 301) {
-    if ([request.URL isEqual:[self serverURLForTarget:kGDTCORTargetFLL]]) {
-      NSURLRequest *newRequest = [self constructRequestForTarget:kGDTCORTargetCCT
-                                                            data:task.originalRequest.HTTPBody];
-      completionHandler(newRequest);
-    }
+    // TODO: Take a redirect URL from the response.
```

Comment:
Is this intended to be commented?

Reply:
Yes, it needs to be slightly updated to make sure any redirect is handled correctly, not only from the specific backend.